Detectors/ITSMFT/ITS/workflow/src/ThresholdCalibratorSpec.cxx

```diff
@@ -954,6 +955,7 @@ void ITSThresholdCalibrator::addDatabaseEntry(
 
           // Noisy pixel IDs
           pixIDs_Noisy += std::to_string(v[i]);
+          nNoisy++;
```

Comment:
Do you need `nNoisy`? It should be the same as `n_pixels = v.size()` when the pixel type is "noisy" right?